ignite/distributed/utils.py

```diff
@@ -426,6 +427,14 @@ def barrier() -> None:
     _model.barrier()
 
 
+def new_group(ranks: List[int], **kwargs: Any) -> Any:
+    """Helper method to make group for each backend from ranks."""
```

Comment:
One last thing before merging. We need to update the docstring. See for example the one for `broadcast`. As you have input args, kwargs we need to document that. In the end you need to add 
```
.. versionadded:: 0.5.0
```